src/librbd/cache/pwl/ssd/LogEntry.cc

```diff
@@ -24,7 +24,11 @@ buffer::list& WriteLogEntry::get_cache_bl() {
   std::lock_guard locker(m_entry_bl_lock);
   return cache_bl;
 }
-
+void  WriteLogEntry::copy_cache_bl(bufferlist *out) {
+  ceph_assert(out);
```

Comment:
The assert looks unnecessary to me -- we would get a clear segfault on `*out` below.